components/Tabs/Tabs.js

```diff
@@ -2,50 +2,54 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+     this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = this.element.dataset.tab;
     
-    // Using the custom data attribute get the associated Item element
-    // this.itemElement;
     
+    // Using the custom data attribute get the associated Item element
+     this.itemElement = document.querySelector(`.tabs-item[data-tab='${this.data}']`);
+     console.log(this.itemElement)
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+     this.tabItem = new TabItem(this.itemElement);
+     console.log(this.tabItem)
+     
     
     // Add a click event listener on this instance, calling the select method on click
-
+    this.element.addEventListener('click', () => this.select())
   };
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
+     let links = document.querySelectorAll('.tabs-link')
 
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
+   links.forEach(link => link.classList.remove('tabs-link-selected'))
+  
     
+    // Add a class named "tabs-link-selected" to this link
+    this.element.classList.add('tabs-link-selected');
+    //console.log(this.data)
     // Call the select method on the item associated with this link
-
+    this.tabItem.select();
   }
 }
 
 class TabItem {
   constructor(element) {
     // Assign this.element to the passed in element
-    // this.element;
+    this.element = element;
   }
 
   select() {
     // Select all ".tabs-item" elements from the DOM
-    // const items;
+    const items = document.querySelectorAll('.tabs-item');
 
     // Remove the class "tabs-item-selected" from each element
-    
+   Array.from(items).forEach(item => item.classList.remove('tabs-item-selected'))
```

Comment:
if you're doing a forEach, no need to convert it to an Array.  forEach works on nodelists.